app/assets/javascripts/common/reports.js

```diff
@@ -1669,6 +1569,85 @@ Reports = function (withLtfu) {
   };
 };
 
+// Base config object which can be built on using combineConfigWithBaseConfig(AdditionalConfigObject)
+// AdditionalConfigObject is used to add or overwrite specfic key: value - [will take priority]
+
+function baseLineGraphConfig() {
```

Comment:
Consider `baseGraphConfig`, since we "combine with base" in our methods, not "combine with baseline"

Reply:
The idea for this name is that this is the "base - line graph - config" rather than "baseline".
My intention is to explore splitting the config for line graphs and bar charts for simplified modifications in future.

Reply:
Ohhhh that makes perfect sense. Neato.

Reply:
If that's the case, let's perhaps rename the `withBaseConfig()` method to `withBaseLineConfig()`, since that's the one that this method adds on. We can then add `withBaseGraphConfig()` later as well. Does that make sense?

Reply:
Changes done 👍